addons/test_mail/tests/test_message_post.py

```diff
@@ -48,7 +48,12 @@ def setUpClass(cls):
             'name': 'Test',
             'email_from': 'ignasse@example.com'
         })
+        cls.test_record_main_attachment = cls.env['mail.test.simple.main.attachment'].with_context(cls._test_context).create({
```

Comment:
As it is used in a single test can't we move it directly into the test itself ?

Reply:
Yes good idea , thanks.